public_html/wp-content/plugins/wcpt/stubs/page/code-of-conduct-online.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/** @var WordCamp_New_Site $this */
+
+?>
+<!-- wp:paragraph {"customBackgroundColor":"#eeeeee"} -->
+<p style="background-color:#eeeeee" class="has-background"><em>Organizers note:</em> Below is a boilerplate code of conduct that you can customize; another great example is the Ada Initiative <a href="http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy">anti-harassment policy.</a></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"customBackgroundColor":"#eeeeee"} -->
+<p style="background-color:#eeeeee" class="has-background">We also recommend the organizing team read this article on <a href="http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports">how to take a harassment report</a></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"customBackgroundColor":"#eeeeee"} -->
+<p style="background-color:#eeeeee" class="has-background">Please update the portions <span style="color: red; text-decoration: underline;">with red text.</span> To remove the red underline formatting, click the arrow in the toolbar called "more rich text controls," and toggle off the underline.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"level":3} -->
+<h3>1. Purpose</h3>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p><span style="color: red; text-decoration: underline;">WordCamp YourCityName</span> believes our community should be truly open for everyone. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, disability, ethnicity, religion, preferred operating system, programming language, or text editor.</p>
```

Comment:
```suggestion
<p><span style="color: red; text-decoration: underline;">WordCamp YourCityName</span> believes our community should be truly open for everyone. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, disability, ethnicity, religion, age, caste, social class, preferred operating system, programming language, or text editor, among other identifying characteristics.</p>
```

Reply:
I think it would be best to make this change as a separate, standalone PR. That way if people want to link to the changeset for some reason, it won't be buried in other unrelated changes.